projects/aca-content/src/lib/ui/theme.scss

```diff
@@ -18,7 +22,7 @@ mat-icon {
   color: var(--theme-secondary-text);
 }
 
-#{$mat-text-field-filled}:not(#{$mat-text-field-disabled}) {
+#{$mat-text-field-filled}:not(#{$mat-text-field-disabled})#{$mat-form-field-wrapper} {
```

Comment:
Can you move the #{$mat-form-field-wrapper} selector to before the :not() segment. Otherwise it gets a bit complicated to read the whole selector